azure-pipelines.yml

```diff
@@ -0,0 +1,87 @@
+trigger:
+  branches:
+    include: ['*']
+  tags:
+    include: ['*']
+
+resources:
+  repositories:
+    - repository: templates
+      type: github
+      name: xoac/rust-azure-pipelines
+      ref: refs/heads/master ## TODO you may want to change it to refs/tags/TAG_NAME.
```

Comment:
You probably wanna change it from master to `/tags/v0.0.2` since I can break master any time

Reply:
@chmln (just to make sure you will read this)